doc/en/customize.rst

```diff
@@ -97,6 +97,19 @@ check for ini-files as follows::
 .. _`how to change command line options defaults`:
 .. _`adding default options`:
 
+Change which files are added to sys.path
+-----------------------------------------------
+
+If you place a conftest.py file in the root directory of your project
+(as determined by pytest, see above.) pytest will run tests against 
+the code below that directory by adding it to your sys.path instead of 
```

Comment:
This is kind of a hack, which admittedly I've used and even advertised in the past.  But it's a side-effect on the not very ideal way how we import conftest.py files currently.  I'm rather hesitant to document it officially as then it becomes something we can not break in the future, while this is a domain where it would be very nice to clean up these side-effects.  We shouldn't be doing such unexpected things with sys.path!